tls/vibe/stream/openssl.d

```diff
@@ -1504,7 +1495,7 @@ private void setSSLError(string msg, string submsg, int line = __LINE__, string
 	ERR_add_error_data(3, msg.toStringz, ": ".ptr, submsg.toStringz);
 }
 
-static if (OPENSSL_VERSION.startsWith("1.1")) {
+static if (OPENSSL_VERSION_AT_LEAST(1, 1, 0)) {
```

Comment:
Note that this is probable very broken on OpenSSL 3.0 as a result, because it uses a (most likely outdated) internal struct definition.